test/WebJobs.Script.Tests/ContentEndToEndTests.cs

```diff
@@ -19,9 +19,9 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Tests.Content
 {
-    public class NodeTextPlain : ContentTests
+    public class NodeTextPlain : ContentTests<NodeContentFixture>
```

Comment:
Is there a reason why the TextPlain and ApplicationJson test groups are in two different test suite classes, as opposed to one? The reason I ask is because it means we incur the ScriptHost startup costs multiple times unecessarily. I try to minimize this when possible, because all the hosts contribute to our test flakiness.